splinepy/multipatch.py

```diff
@@ -154,6 +199,29 @@ def boundaries(self):
 
         return boundary_list
 
+    @property
+    def boundary_patches(self):
+        """Extract all boundary patches of a given Multipatch system as splines
+
+        Returns
+        -------
+        boundary_patches : list
+          list of all embedded splines
+        """
+        if self._boundary_splines is None:
```

Comment:
This doesn't work well with inplace changes

Reply:
This is true, I am not yet sure how to handle the in-place changes at all, especially if splines are modified on the fly. The only safeguard I have at this point is in the splines-setter, where - if called - all dependent members are reinitialized... I have no better solution at this point

Reply:
I'd say, we can leave this to always recompute or add a `use_saved` param until we can handle in-place changes